src/odemis/driver/simsem.py

```diff
@@ -540,6 +544,9 @@ def _acquire_thread(self, callback):
                 # as in Odemis the convention for SEM is that the ebeam waits
                 # for _all_ the detectors to be ready before scanning.
                 self.data._waitSync()
+                if first_frame:
+                    self.parent._scanner.startScan.notify()
+                    first_frame = False
```

Comment:
Does this automatically stop notifying something which is similar to the unsubscribe used in the testcases when self._acquisition_must_stop.clear() is called in finally: section?

Reply:
No, this is different. It's a model.Event, an Odemis object that works in between Components. `_acquisition_must_stop` is a threading.Event, a standard Python object that only works within the component (because each component runs in a separate Python process).